scripts/setup-ubuntu-20.04.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+apt-get install -y \
```

Comment:
I think this is a complete list. I used to test on a VM and may have forgotten to copy things over. I tried testing in a docker container but ran out of bandwidth.